### DISCUSSION TITLE
Large Gallery view

### DISCUSSION THREAD

**Discussion by martinesmann** [Azure/communication-ui-library#1884] [General]
Is it possible to use the Large Gallery view feature available in Teams? 
Both the desktop and web version of Teams support "Large Gallery" view.
The Large Gallery view allows for viewing 2 x 49 participant videos simultaneously.

How can this this feature be used with Communication UI Library and or Azure Communication SDK?

**Comment by ddematheu2** [comment]
Hey @martinesmann no current support for Large Gallery view. ACS only supports up to a 2x2 video grid: https://docs.microsoft.com/en-us/azure/communication-services/concepts/voice-video-calling/calling-sdk-features#calling-sdk-streaming-support

**Reply by martinesmann** [reply]
When can we expect support for Large Gallery view? 
My impression is that ACS is aiming to be on-pair with the feature set and capabilities available to the web version of teams, right?

**Reply by martinesmann** [reply]
@[ddematheu2](https://github.com/ddematheu2) do you have any information to share on when we can expect the Large Gallery feature to be available?

**Comment by martinesmann** [comment]
Any update on when this would be supported... on the roadmap? 
@ddematheu2

**Comment by martinesmann** [comment]
Anyone who can answer or help here... question has been hanging for 2 weeks now...  
@prprabhu-ms @JamesBurnside @ddematheu2 @mgamis-msft @anjulgarg @dmceachernmsft @edwardlee-msft @carocao-msft

**Comment by ddematheu2** [comment]
Hey @martinesmann apologies for the late response. This item is currently not on a defined roadmap.

**Reply by martinesmann** [reply]
Hi @ddematheu2 
Thanks for the follow-up. Is there something I/the community/customers/users can do to motivate/drive this feature to be implemented? 
As an example I'm building a solution targeting lung patients and people with reduced lung capacity... they don't have the same degree of freedom to move around and meet people... ACS and Communication Library allows them to build an online community and they use it and are building new friendships and relations... cross county and local...  but the limit to only see 4 people in a group is not ideal and it would be a real game changer if we could get more video feeds in the call... it would stregthen the community and allow it to grow even more.

**Comment by alkwa-msft** [comment]
Closing this discussion due to inactivity. Feel free to re-open if this is still a topic we would like to continue.
